engineering/conventions-go.md

```diff
@@ -62,4 +57,10 @@ A blank line is usually added after the end of any control structure, too.
 
 ### Naming
 
-* Prefer full names (e.g. `Repository`, not `Repo`).
\ No newline at end of file
+* Prefer full names (e.g. `Repository`, not `Repo`).
+
+## CLI
+
+* Put sources for your executable commands in `cmd/<command-name>/`, with the `main` function in `cmd/<command-name>/main.go`.
```

Comment:
Even when there's only one executable?

See `pga` as an example of something done the other way around.
No strong opinion, just wondering whether we need to retroactively apply this rule (which I had totally missed before)

Reply:
Yes, we do this even for single executables. I guess we don't really need to go wild on retroactivity. For example, [rovers](https://github.com/src-d/rovers/) does not follow this, it uses the top level directory for the `main`, which is what we used to do in the past. But we'll probably change it at some point.